cypress/integration/search_page_spec.js

```diff
@@ -0,0 +1,191 @@
+const paginator = '[data-testid=paginator]'
+const dialogContent = '[data-testid=dialog-content]'
+const search = '[data-testid=search]'
+const datasets = '[data-testid=dataset-list]'
+const firstDataset = '[data-testid=dataset-list] > :nth-child(1)'
+const apiAccessibleCheckbox = '[data-testid=api-accessible-checkbox]'
+const organizations = '[data-testid=facet-sidebar] > :nth-child(1)'
+const cogoCheckBox = '[data-testid=facet-sidebar] > :nth-child(1) > :nth-child(2) > .checkbox-indicator'
+const keywords = '[data-testid=facet-sidebar] > :nth-child(2)'
+const bicycleCheckBox = '[data-testid=facet-sidebar] > :nth-child(2) > :nth-child(2) > .checkbox-indicator'
+
+const routes = {
+  allDatasetsLastModified: {
+    method: 'GET', 
+    url: '/api/v1/dataset/search?offset=0&limit=10&sort=last_mod&query=&apiAccessible=true',
+    response: 'fixture:search_page_spec/all_datasets_last_modified'
+  },
+  allDatasetsNameAsc: { 
+    method: 'GET', 
+    url: '/api/v1/dataset/search?offset=0&limit=10&sort=name_asc&query=&apiAccessible=true', 
+    response: 'fixture:search_page_spec/all_datasets_name_asc'
+  },
+  allDatasetsNameDesc: {
+    method: 'GET', 
+    url: '/api/v1/dataset/search?offset=0&limit=10&sort=name_desc&query=&apiAccessible=true',
+    response: 'fixture:search_page_spec/all_datasets_name_desc'
+  },
+  allDatasetsPage2: {
+    method: 'GET',
+    url: '/api/v1/dataset/search?offset=10&limit=10&sort=name_asc&query=&apiAccessible=true',
+    response: 'fixture:search_page_spec/all_datasets_page_2'
+  },
+  apiAccessibleFalseDatasets: {
+    method: 'GET',
+    url: '/api/v1/dataset/search?offset=0&limit=10&sort=name_asc&query=&apiAccessible=false',
+    response: 'fixture:search_page_spec/apiAccessible_false_datasets'
+  },
+  bicycleDatasets: {
+    method: 'GET',
+    url: '/api/v1/dataset/search?offset=0&limit=10&sort=name_asc&query=&facets[keywords][]=bicycle&apiAccessible=true',
+    response: 'fixture:search_page_spec/bicycle_datasets'
+  },
+  cogoDatasets: {
+    method: 'GET',
+    url: '/api/v1/dataset/search?offset=0&limit=10&sort=name_asc&query=&facets[organization][]=COGO&apiAccessible=true',
+    response: 'fixture:search_page_spec/cogo_datasets'
+  },
+  cotaDatasets: {
+    method: 'GET',
+    url: '/api/v1/dataset/search?offset=0&limit=10&sort=name_asc&query=COTA&apiAccessible=true',
+    response: 'fixture:search_page_spec/cota_datasets'
+  },
+  info: {
+    method: 'GET',
+    url: '/sockjs-node/*',
+    response: 'fixture:search_page_spec/info.json'
+  },
+  ogripDataset: {
+    method: 'GET',
+    url: '/api/v1/organization/ogrip/dataset/622746a5_4e2a_4a4c_ac18_74cb1fb05ab3',
+    response: 'fixture:search_page_spec/ogrip_dataset'
+  }
+}
+
+function isDefaultPage () {
+  cy.get(organizations).children('.checkbox').should('have.length', 8)
+  cy.get(keywords).children('.checkbox').should('have.length', 10)
+  cy.get(datasetsFoundCount).contains('19 datasets found')
```

Comment:
It would be nice to tie some of these strings back to the fixture data in a more dynamic/less brittle way.

Reply:
We might consider something like https://www.npmjs.com/package/cypress-autorecord in the future

Reply:
good idea